packages/common/src/content/_fetch.ts

```diff
@@ -94,6 +110,7 @@ const getHubEnrichmentsOptions = (
   const opts = cloneObject(requestOptions);
   opts.params = {
     ...opts.params,
+    // TODO: we should fetch errors too
```

Comment:
I stopped short of including `errors` in the enrichments we fetch from the Hub API, but we should do that. I'll add it in a follow on PR